CHANGELOG.md

```diff
@@ -213,7 +213,11 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [I18n] Fix Listr type errors and error handlers ([#3629](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3629))
 - [Multiple DataSource] Refactor dev tool console to use opensearch-js client to send requests ([#3544](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3544))
 - [Multiple DataSource] Present the authentication type choices in a drop-down ([#3693](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3693))
+- [Console] Remove unused ul element and its custom styling ([#3993](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3993))
```

Comment:
Seems you include ur other PRs. CHANGELOG should only consider your current PR. For example, you should only include #4148. This might cause some future merge conflicts. For example if you merge #3798 first, then you would need to solve the conflict before merging this one. Including changes that are unrelated to the PR's can lead to confusion to others.